nhais/README.md

```diff
@@ -0,0 +1,40 @@
+# NHAIS Adaptor - Developer Notes
+The following sections are intended to provide the necessary info on how to configure and run the NHAIS adaptor.
+
+## Pre-requisites
+
+Ensure you have Pipenv installed and on your path, then within nhais directory, run:
+
+`pipenv install`
+
+## Developer setup 
+
+Start IntelliJ
+
+Install Python and EnvFile plugin
+
+Restart IntelliJ
+
+Open nhais component - IntelliJ → File → Open → select nhais folder (../integration-adaptors/nhais)
+
+Open main.py file
+
+You’ll see information that there’s “No Python interpreter configured“
```

Comment:
I did not have such an information when configuring mhs. What I needed to do was going to File -> Project Structure -> SDK -> Add new Python SDK and provide a path to the executable of my pipenv.

Reply:
I believe either method is fine, this might be clearer